Platform/RaptorlakeBoardPkg/Rplp/Fsp/FspBinRplp.inf

```diff
@@ -0,0 +1,28 @@
+## @file
+#  File to describe FSP repo information
+#
+#  Copyright (c) 2020 - 2024, Intel Corporation. All rights reserved.<BR>
+#
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+#
+##
+
+
+[UserExtensions.SBL."CloneRepo"]
+  REPO    = https://github.com/intel/FSP.git
+  COMMIT  = fc5e3c9aa6779d8d83548dc0ff397843491d2b1b
```

Comment:
FSP repo has a new commit updating VBT json file.
https://github.com/intel/FSP/commit/98e497f733450b97f590cab0556f05d23cdf3e02

Reply:
Updated commit and removed vbt files.